packages/blade/src/components/Link/Link/__tests__/Link.web.test.tsx

```diff
@@ -26,16 +26,22 @@ describe('<Link />', () => {
     expect(container).toMatchSnapshot();
   });
 
-  it('should render link with an href, target and rel', () => {
+  it('should render link with an href, target, rel and title', () => {
     const linkText = 'Learn More';
     const { getByRole } = renderWithTheme(
-      <Link href="https://www.google.com/" target="_blank" rel="noreferrer noopener">
```

Comment:
add to the default story as well?

Reply:
Missed adding it here. Created new PR https://github.com/razorpay/blade/pull/1116